content/en/docs/distributions/aws/deploy/eks-compatibility.md

```diff
@@ -34,4 +34,9 @@ Starting with Kubeflow version 1.2, Amazon EKS maintains end-to-end testing betw
 </div>
 
 - **Incompatible**: the combination is not known to work together
-- **Compatible**: all Kubeflow features have been tested and verified for the EKS Kubernetes version
\ No newline at end of file
+- **Compatible**: all Kubeflow features have been tested and verified for the EKS Kubernetes version
+
+### Kubeflow v1.4 and v1.5 Support
+
+Support for Kubeflow-v1.4.1 and Kubeflow-v1.5.x are in active development. You can track the following issues: [v1.4 tracking issue](https://github.com/awslabs/kubeflow-manifests/issues/27), [v1.5 tracking issue](https://github.com/awslabs/kubeflow-manifests/issues/91) to stay up-to-date on progress.
```

Comment:
Nit, recommendation for clarity: 

You can track the following issues to stay up-to-date on progress:
- [v1.4 tracking issue](https://github.com/awslabs/kubeflow-manifests/issues/27)
- [v1.5 tracking issue](https://github.com/awslabs/kubeflow-manifests/issues/91)